test/test_helper.rb

```diff
@@ -10,7 +10,7 @@
 Mocha.configure do |c|
   c.strict_keyword_argument_matching = true
 end
-Minitest::Reporters.use!
+Minitest::Reporters.use! [Minitest::Reporters::DefaultReporter.new(:color => true)]
```

Comment:
This looks unrelated. Can this be separated, at least in a different commit?

Reply:
Done.